tests/scryer/cli/fixtures/toplevel_test_helper.pl

```diff
@@ -0,0 +1,50 @@
+% Helper predicates for testing custom toplevel functionality
+
+success_toplevel :-
+    write('SUCCESS_TOPLEVEL_EXECUTED'), nl,
```

Comment:
`format/2` and strings seem a bit more suitable for such ephemeral data, as in:

<pre>
<b>format("SUCCESS_...~n", [])
</pre>